AppiumForMac/Library/CocoaHTTPServer/Vendor/CocoaLumberjack/DDFileLogger.m

```diff
@@ -416,7 +416,7 @@ - (NSString *)createNewLogFile
 	NSString *logsDirectory = [self logsDirectory];
 	do
 	{
-		NSString *fileName = [NSString stringWithFormat:@"log-%@.txt", [self generateShortUUID]];
+		NSString *fileName = [NSString stringWithFormat:@"AFMlog.txt"];
```

Comment:
This changes the whole purpose of this function. How about collisions?

Reply:
Its creating new file every time instead of appending to existing file due to random udid. I just kept static file so that it will append to the existing file.